### DISCUSSION TITLE
Please help with External Hard Drive Access

### DISCUSSION THREAD

**Discussion by shruru** [rxhanson/Charmstone-Community#15] [Q&A]
Hi there, I really like this App and purchased in App Store for full version. (Sorry for my english)
However I just found it can't access to external hard driver 's folder every time restart the Mac. (If failed)

well, initally, If I add a shortcut , it works very well, but after restart Mac, the folder's icon becames blank and alert:
Can't access to this location. I have to re-create it and then it works again...

The external hard drive is APFS+encryption mode. (however the password already saved in KeyChain)

https://imgur.com/Sg6r3E0

**Comment by rxhanson** [comment]
I added some retry logic into the latest version that is now released, v0.18. Please updated to it and let me know if it fixes the issue for you.
